ocs_ci/deployment/deployment.py

```diff
@@ -179,41 +179,55 @@ def label_and_taint_nodes(self):
         Label and taint worker nodes to be used by OCS operator
         """
 
+        arbiter_deployment = config.DEPLOYMENT.get("arbiter_deployment")
+
         nodes = ocp.OCP(kind="node").get().get("items", [])
         worker_nodes = [
             node
             for node in nodes
-            if "node-role.kubernetes.io/worker" in node["metadata"]["labels"]
+            if constants.WORKER_LABEL in node["metadata"]["labels"]
         ]
         if not worker_nodes:
             raise UnavailableResourceException("No worker node found!")
         az_worker_nodes = {}
         for node in worker_nodes:
-            az = node["metadata"]["labels"].get(
-                "failure-domain.beta.kubernetes.io/zone"
-            )
+            az = node["metadata"]["labels"].get(constants.ZONE_LABEL)
```

Comment:
@iamniting we do not get this label for on-prem cluster.

IIUC, the label which gets added as part of OCS install is as following. We need to handle this

compute-0         Ready    worker   19d   v1.19.0+7070803   app.kubernetes.io/name=vault,beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,cluster.ocs.openshift.io/openshift-storage=,kubernetes.io/arch=amd64,kubernetes.io/hostname=compute-0,kubernetes.io/os=linux,node-role.kubernetes.io/worker=,node.openshift.io/os_id=rhcos,**topology.rook.io/rack=rack0**

Reply:
Hi Neha, as discussed on the chat, labels are one of the prereqs for the arbiter cluster. So we are assuming the admin will add those labels when there are no labels present beforehand.

I have added support for the `failure-domain.beta.kubernetes.io/zone` and `topology.kubernetes.io/zone` both of them.

Reply:
ack